### DISCUSSION TITLE
how to do it in NPM?

### DISCUSSION THREAD

**Discussion by MrColumbo** [NginxProxyManager/nginx-proxy-manager#3650] [Q&A]
I am struggle to do something which actually should be quite easy .....basically I am using immich for a while and it is working 95% quite well with no special setting but when I upload files larger than 500 MB it does not work while it work in the local network when i bypass npm.

There are some instructions for nginx here :https://immich.app/docs/administration/reverse-proxy/

...but i am not quite sure which of those settings are default by npm and which needs to be done and how. I guess the client max size is the key element. 

I guess there are a lot of users who use immich and NPM as well and already did that :)

**Comment by Vaadasch** [comment]
The link you provided ask to add client_max_body_size 50000M; to the server block on nginx.
Did you search "client_max_body_size nginx proxy manager" on google ?
